### ISSUE TITLE
Where do I get Weather_Radar_Stations.geojson?

### ISSUE THREAD

**Issue by ahuang11** [jthielen/OpenMosaic#5]
I am looking at https://github.com/jthielen/OpenMosaic/blob/main/examples/process_full_conus.py
```
    parser.add_argument(
        "-n",
        "--nexrad-file",
        help="NEXRAD Site GeoJSON file",
        default="~/Weather_Radar_Stations.geojson",
        metavar="~/Weather_Radar_Stations.geojson"
    )
```
Do I need it if I already have .bz2 files from https://nomads.ncep.noaa.gov/pub/data/nccf/radar/nexrad_level2/

**Comment by jthielen** [comment]
Good catch! That file is from [the HIFLD platform](https://hifld-geoplatform.opendata.arcgis.com/datasets/weather-radar-stations), and a direct download is available at this link: https://opendata.arcgis.com/datasets/85c59fe951504e9b9919e24d7a684084_3.geojson

Definitely a good thing to get into the documentation (or to provision with pooch if other data files are also needed locally going forward)!
